Makefile

```diff
@@ -155,6 +155,15 @@ build: fmt vet ## Build manager binary.
 		" \
 		-o bin/manager main.go
 
+.PHONY: docker-build
+docker-build: fmt vet ## Build manager binary.
```

Comment:
can we rename this directive. This is a little too similar to `make image-build` which builds the container image and I can see it possibly causing a bit of confusion. Something like `go-build-for-image` maybe, WDYT?